internal/pkg/block/block_test.go

```diff
@@ -54,7 +54,10 @@ func TestTriangleEncoding(t *testing.T) {
 		types.AssertHaveSameCid(t, exp, &cborJSONRoundTrip)
 	}
 	t.Run("encoding block with zero fields works", func(t *testing.T) {
-		testRoundTrip(t, &blk.Block{})
+		testRoundTrip(t, &blk.Block{
+			BlockSig:        crypto.Signature{Type: crypto.SigTypeSecp256k1, Data: []byte{}},
+			BLSAggregateSig: crypto.Signature{Type: crypto.SigTypeBLS, Data: []byte{}},
```

Comment:
New specs-actors signature encoding requires that signatures be non-nil.

Reply:
This requirement will be removed if we merge: https://github.com/filecoin-project/specs-actors/pull/223 which will not error when unmarshalling zero valued signatures.

Reply:
FWIW I don't think it's a bad requirement, and would prefer we move toward maintaining it.